l10n_eu_nace/i18n/ru.po

```diff
@@ -0,0 +1,5236 @@
+# Translation of OpenERP Server.
```

Comment:
This file is broken: https://travis-ci.org/github/OCA/community-data-files/jobs/701976078#L1278-L3835

Please delete it for the bot to be able to continue with the merge.